account_analytic_default_product_category/__manifest__.py

```diff
@@ -3,16 +3,17 @@
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 {
     'name': 'Account Analytic Default Product Category',
-    'summary':"""""",
+    'summary': """""",
     'version': '10.0.1.0.0',
     'category': 'Accounting & Finance',
-    'description': """ 
+    'description': """
 * Add 'Analytic Account' to Product Category
-* If no corresponding Analytic Defaults, automatically propose Analytic Account of Product Category on Invoice lines
+* If no corresponding Analytic Defaults, automatically propose
+Analytic Account of Product Category on Invoice lines
 * Do not propose it on Purchase Order lines
     """,
-    'author': 'Rooms For (Hong Kong) Limited T/A OSCG',
-    'website': 'https://www.odoo-asia.com',
+    'author': 'Rooms For (Hong Kong) Limited T/A OSCG, Quartile Limited',
```

Comment:
```suggestion
    'author': 'Quartile Limited',
```
Rooms For (Hong Kong) Limited is our old company name.

Reply:
Yes. I thought that is our parent company and I was not sure removing our parent company from author list. I will remove it now.